packages/nitro-protocol/test/contracts/NitroAdjudicator/concludePushOutcomeAndTransferAll.test.ts

```diff
@@ -1,8 +1,8 @@
 import {expectRevert} from '@statechannels/devtools';
-import {Contract, Wallet, ethers, BigNumber} from 'ethers';
+import {Contract, Wallet, ethers, BigNumber, utils, constants} from 'ethers';
 
-import AssetHolderArtifact1 from '../../../artifacts/contracts/test/TESTAssetHolder.sol/TESTAssetHolder.json';
-import AssetHolderArtifact2 from '../../../artifacts/contracts/test/TESTAssetHolder2.sol/TESTAssetHolder2.json';
+import EthAssetHolderArtifact1 from '../../../artifacts/contracts/test/TestEthAssetHolder.sol/TestEthAssetHolder.json';
+import EthAssetHolderArtifact2 from '../../../artifacts/contracts/test/TestEthAssetHolder2.sol/TestEthAssetHolder.json';
```

Comment:
Ok, now I think I understand. The diff made me think there was a rename happening, combined with other changes. (There is a rename, as `AssetHolder1` is renamed to `EthAssetHolder1`.)

But this change is _not_ a rename, and you are importing

8b173676315f947ae8f9730b5365ea03526dcaa3 would have been more clear if it were written something like

```
use TestEthAssetHolder in pushOutcomeAndTransferAll test

The TESTAssetHolder that is currently used is not a real asset holder,
since it doesn't send funds to the destination. This is because it
extends the base AssetHolder contract.

We switch to TestEthAssetHolder & TestEthAssetHolder2 to
get a more accurate measure of gas consumption.
```